MovingBlocks/DestinationSol
# Refactor speed and velocity

# Description

Refactors usage of `speed` to `velocity` where vector variables were labeled as `speed` and  `speedLen` to `speed`. Also keeps original pre/su-fixes of such variables.

# Testing

- Search for `speed` occurences in codebase, they should refer to scalar variables only
- Search for `speedLen` and `velocityLen`, there should be none
- Search for `velocity`, should refer only to Vector2 objects
- Start the game, play it for a while, nothing should change from the play perspective

# Outstanding Work

Nothing right now.

# Notes
Commits are made per package refactoring, so some continuity could be accomplished during refactoring, but it is intended to be squashed before merging.

Review required @eviltak

Fixes #290
